covid_graphs/covid_graphs/formula.py

```diff
@@ -107,40 +108,82 @@ def formula(x: float):
         )
 
 
+def set_proto_date(proto_date, date: datetime.date) -> None:
```

Comment:
Nit: Usual convention is to prepend functions not exposed outside of the file with `_`. (There are a few more below.)

Reply:
Done